tests/functional/confirm.js

```diff
@@ -88,7 +88,7 @@ define([
           .click()
         .end()
 
-        .findByCssSelector('#open-gmail')
+        .findByCssSelector('.gmail')
```

Comment:
@larissagaulia see if you can use the https://www.w3.org/TR/selectors/#attribute-selectors here using the data attribute. some other tests might already have that in place.